app/.htaccess

```diff
@@ -278,7 +278,7 @@ AddDefaultCharset utf-8
 # see the specification: http://w3.org/TR/CSP).
 
 # <IfModule mod_headers.c>
-#    Header set Content-Security-Policy "script-src 'self'; object-src 'self'"
+#    Header set Content-Security-Policy "script-src 'self' https://accounts.google.com/gsi/client; object-src 'self'"
```

Comment:
Does this change do anything? Isn't this entire line just a comment?

Reply:
I guess it doesn't do anything. But if the CSP ever gets used, it will need to be there.